schema/postgres/1494804546_create_repositories_table.up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE EXTENSION IF NOT EXISTS pgcrypto;
+
+CREATE TABLE repositories (
+  id             UUID PRIMARY KEY                           DEFAULT gen_random_uuid(),
+  name           VARCHAR(100) NOT NULL,
```

Comment:
can just be `TEXT NOT NULL`, really, for consistency

https://www.depesz.com/2010/03/02/charx-vs-varcharx-vs-varchar-vs-text/